README.md

```diff
@@ -8,8 +8,10 @@ Monitors your Elasticsearch cluster and reports metrics to CloudWatch.
 * Export the following variables, adapting them for your account (this is for Deploy Tools)
 
 ```bash
-export TagQueryApp="elk-es-master"
-export TagQueryStack="deploy"
-export ClusterName="elk"
+export TAG_QUERY_APP="elk-es-master"
+export TAG_QUERY_STACK="deploy"
+export CLUSTER_NAME="elk"
 ```
-* Then run it by using `sbt run`
+
+* Run `python3 -m http.server 8000 --directory test-data` to serve static test data
```

Comment:
I think most developers will have this as it seems to be installed by Xcode Command Line Tools, which is set up as part of https://github.com/MikeMcQuaid/strap?tab=readme-ov-file#features.